srcpkgs/frp/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'frp'
 pkgname=frp
-version=0.37.1
-revision=3
+version=0.58.1
+revision=1
 build_style=go
 go_import_path=github.com/fatedier/frp
 go_package="${go_import_path}/cmd/frpc ${go_import_path}/cmd/frps"
 short_desc="Fast reverse proxy"
-maintainer="Anachron <gith@cron.world>"
+maintainer="Suyun <suyun@aosc.io>"
```

Comment:
revert this line, @Anachron is still the maintainer

Reply:
i am sorry for the misunderstanding